updating/understanding-upgrade-channels-release.adoc

```diff
@@ -6,7 +6,17 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-Update channels are tied to a minor version of {product-title}. For instance, {product-title} 4.10 update channels recommend updates to 4.10 and updates within 4.10. They also recommend updates within 4.9 and from 4.9 to 4.10 allowing all 4.9 to eventually update to 4.10, even if they do not immediately meet the minimum z-stream version requirements. They do not recommend updates to 4.11 or later releases. This strategy ensures that administrators explicitly decide to update to the next minor version of {product-title}.
+Update channels are the mechanism by which users declare the {product-title} minor version they intend to update to. The Cluster Version Operator uses an update graph based on the channel declaration, along with other conditional information, to provide a list of recommended updates available to the cluster.
+
+Update channels correspond to a minor version of {product-title}. The version number in the channel represents the target minor version that the cluster will eventually be updated to, even if it is higher than the cluster's current minor version.
+
+For instance, {product-title} 4.10 update channels provide the following recommendations:
+
+* Updates within 4.10
+* Updates within 4.9
+* Updates from 4.9 to 4.10, allowing all 4.9 clusters to eventually update to 4.10, even if they do not immediately meet the minimum z-stream version requirements.
```

Comment:
And, with `eus-4.10`, also updates from 4.8 to 4.9 to 4.10, and updates within 4.8.  Not sure how to square wanting to call that out for 4.even releases without making the docs awkward to maintain for 4.odd release.

Reply:
I can consider adding two more bullet points and beginning each with something like "(only eus-4.10)". 

Also, assuming I'm understanding your point correctly: if this is just an example of how channels work, does the specific version number in the example need to be so recent as to have those maintainability issues? I feel like the 4.13 version of this page, for instance, might look weird with these references to the 4.10 channels, but the accuracy and education of the content is still valid, right?